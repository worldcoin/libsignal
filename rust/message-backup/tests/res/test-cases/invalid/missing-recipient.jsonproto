[
  {
    "version": "1",
    "backupTimeMs": "1705692409729"
  },
  {
    "recipient": {
      "id": "1",
      "self": {}
    }
  },
  {
    "chat": {
      "id": "1",
      "recipientId": "2"
    }
  }
]