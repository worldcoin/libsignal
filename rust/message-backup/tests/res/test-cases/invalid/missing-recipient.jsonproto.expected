chat frame ChatId(1) error: no record for RecipientId(2)